### DISCUSSION TITLE
Setting a form input value using JS? Not being sent to backend

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7572] [Q&A]
I'm trying to have JS fill in a form field value, and have that sent in my Form.

This is the JS code I'm using
```
document.querySelector('input[name=score]').value = 50
```

I can confirm that the JS is working, however the value is being lost when submitting the form.
Is there any other step that needs to be done to achieve this?

**Comment by edalzell** [comment]
Is `score` field in the Form blueprint?

**Reply by notflip** [reply]
Yes it is. I've tried setting it to "text" or to "hidden". Should it work out of the box when the form input is in the blueprint?
